src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -336,6 +336,11 @@ public static class ConfigurationKeys
         /// </summary>
         public const string JaegerExporterAgentPort = "OTEL_EXPORTER_JAEGER_AGENT_PORT";
 
+        /// <summary>
+        /// Configuration key for specifing vendor plugin path
+        /// </summary>
+        public const string VendorPluginName = "OTEL_VENDOR_PLUGIN";
```

Comment:
1. `VENDOR` prefix does not make sense. Maybe someone would create some common, reusable plugins. 
2. We should agree on some .NET AutoInstr specific prefix. E.g. `OTEL_DOTNET_TRACER`. We do not want to use the same env vars for plugins for e.g. .NET and Java 😉 

I propose to rename this env var to: `OTEL_DOTNET_TRACER_PLUGINS`

Reply:
I like the `OTEL_DOTNET` prefix to help reduce the conflict with other plugin lists. It would be worth reviewing other env vars to improve consistency.